unittests/rewritetests.py

```diff
@@ -44,9 +44,23 @@ def rewrite_raw(self, directory, args):
             if 'MESON_SKIP_TEST' in p.stdout:
                 raise unittest.SkipTest('Project requested skipping.')
             raise subprocess.CalledProcessError(p.returncode, command, output=p.stdout)
+        
         if not p.stderr:
             return {}
-        return json.loads(p.stderr)
+        
+        # Filter out warnings from the output
```

Comment:
What warnings are we concerned about here?

Reply:
Ws still EncodingWarnings. Solved with previous isue.